src/main/flight/rpm_filter.c

```diff
@@ -143,7 +143,8 @@ FAST_CODE_NOINLINE void rpmFilterUpdate(void)
             weight *= rpmFilter.weights[harmonicIndex];
 
             // update notch
-            biquadFilterUpdate(template, frequencyHz, rpmFilter.looptimeUs, rpmFilter.q, FILTER_NOTCH, weight);
+            float dt_compensation = schedulerGetCycleTimeMultiplier();
```

Comment:
You can pull this out of the loop too
```suggestion
            const float dtCompensation = schedulerGetCycleTimeMultiplier();
```

Reply:
Again, not sure this would actually do that by just this change, but yes it should be pulled out of the loop.